package.json

```diff
@@ -47,121 +52,65 @@
     "underscore": "1.7.0"
   },
   "devDependencies": {
+    "autoprefixer": "6.3.6",
     "babel": "6.5.2",
     "babel-cli": "6.7.5",
     "babel-core": "6.7.6",
-    "babel-jest": "11.0.2",
     "babel-loader": "6.2.4",
     "babel-plugin-transform-runtime": "6.7.5",
     "babel-preset-es2015": "6.6.0",
+    "babel-preset-jest": "11.0.2",
     "babel-preset-react": "6.5.0",
+    "babel-preset-react-hmre": "1.1.1",
+    "babel-register": "6.8.0",
     "babel-runtime": "6.6.1",
-    "browser-sync": "2.12.3",
-    "del": "2.2.0",
-    "envify": "3.4.0",
+    "css-loader": "0.23.1",
```

Comment:
`postcss-loader` basically provides you with entry points to run `autoprefixer` etc. Then to load the result we still need `css-loader`.